### ISSUE TITLE
Can I use AVPro trail version on Apple Vision Pro?

### ISSUE THREAD

**Issue by manojahujaTess** [RenderHeads/UnityPlugin-AVProVideo#1912]

**Comment by Chris-RH** [comment]
Hi, the trial version of AVPro Video does support VisionOS.

**Comment by manojahujaTess** [comment]
Thanks for the quick response. 
For some reason, It does not load the video. It works on Editor, but not o the Device/Simulator.

**Comment by Chris-RH** [comment]
1. Have you tried a new project just running AVPro Video demo scenes?
2. What media are you trying to play?
3. Which version of Unity are you using?
4. Which version of AVPro Video are you using?

**Comment by MorrisRH** [comment]
We don't support polyspatial mode at this time, however windowed and full VR work fine.

**Comment by manojahujaTess** [comment]
@Chris-RH  Can you point out where I can find the demo scenes? Thanks

**Comment by manojahujaTess** [comment]
@MorrisRH I am indeed using Polyspatial for my usage. I am using Unbounded settings for it

**Comment by MorrisRH** [comment]
There's a work around for polyspatial mode in this thread: #1800

**Comment by manojahujaTess** [comment]
@MorrisRH @Chris-RH  I have updated my project settings. I am using full VR. I saw the demo scene named InputSystem UI. It looks good but only problem is, I am not able to make Hover work on 2d UI elements.

Some background, I have a world canvas, it has grid layout like system with multi-directional scroll and with buttons, along with masks. I am able to click the buttons but I want some hover effect as well. And also, I am using only Eye interaction and not gaze.

Any help is appreciated. Thanks!
